src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1134,6 +1138,70 @@ public static Collection<FunctionMetadata> ReadFunctionMetadata(ScriptHostConfig
             return functions;
         }
 
+        public Collection<FunctionMetadata> ReadProxyMetadata(ScriptHostConfiguration config, ScriptSettingsManager settingsManager = null)
```

Comment:
It has a unit test. made it internal.

---
In reply to: [133090632](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r133090632) [](ancestors = 133090632)